src/build/render/index.js

```diff
@@ -0,0 +1,78 @@
+const path = require("path");
+const fs = require("fs").promises;
+const R = require("ramda");
+
+const {wrapper} = require("./components");
+const features = require("./features");
+
+async function renderPage(ctx) {
```

Comment:
This is what's left of the page render function after all the features were extracted from it.